docs/topics/which_historical_backend.md

```diff
@@ -13,24 +14,35 @@ Given this variety of backends, the key question is: which one best suits my use
 
 A fifth variable could be whether the backend allows **static data loading**, meaning uploading other datasets independent from the real-time contextual data.
 
-Let’s look at each option from these perspectives.
+Let's analyze these two main options.
+
+## PostgreSQL
```

Comment:
PostGis is described here (great!)

TimeScale (the other main extension we use) should also be included.